.bumpversion.cfg

```diff
@@ -3,8 +3,10 @@ current_version = 0.5.1
 commit = True
 tag = True
 
-[bumpversion:file:doc/source/conf.py]
+[bumpversion:file:doc/source/version.py]
 
-[bumpversion:file:mlos_core/_version.py]
+[bumpversion:file:mlos_core/mlos_core/version.py]
```

Comment:
Make the file non-private to avoid some pylint exceptions and keep it inside the module tree.